src/main/java/http/HttpHeaders.java

```diff
@@ -0,0 +1,15 @@
+package http;
+
+import java.util.HashMap;
+
+public class HttpHeaders extends HashMap<String, String> {
+
+    public static HttpHeaders of(String text) {
+        HttpHeaders headers = new HttpHeaders();
+        for (String line : text.split("\n")) {
+            String[] tokens = line.split(": ");
+            headers.put(tokens[0], tokens[1]);
+        }
+        return headers;
```

Comment:
조합으로 변경하게된다면 아래와 같이 변경해볼 수 있겠네요. 😃

```suggestion
        var headers = Arrays.stream(text.split("\n"))
            .map(line -> line.split(": "))
            .collect(Collectors.toMap(tokens -> tokens[0], tokens -> tokens[1]));

        return new HttpHeaders(headers);
```